src/RefractSourceMap.cc

```diff
@@ -0,0 +1,11 @@
+#include "RefractSourceMap.h"
+
+refract::IElement* drafter::BytesRangeToRefract(const mdp::BytesRange& bytesRange)
```

Comment:
In this case it is wrong definition in `snowcrash` because code snippet above, comes from `MSONSourceMap.h` and should be fixed.